translation/zh_CN.po

```diff
@@ -0,0 +1,1938 @@
+msgstr ""
+"该试验参数使用G10和G11指令来使固件处理回缩。仅在最新的Marlin中支持该功能。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1527
+msgid "Use relative E distances"
+msgstr "使用相对E距离"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1528
+msgid ""
+"If your firmware requires relative E values, check this, otherwise leave it "
+"unchecked. Most firmwares use absolute values."
+msgstr "如果固件需要相对E值，勾选此项，否则不要勾选。大部分固件使用绝对值。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1533
+msgid "Use volumetric E"
+msgstr "使用体积E"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1534
+msgid ""
+"This experimental setting uses outputs the E values in cubic millimeters "
+"instead of linear millimeters. If your firmware doesn't already know "
+"filament diameter(s), you can put commands like 'M200 D[filament_diameter_0] "
+"T0' in your start G-code in order to turn volumetric mode on and use the "
+"filament diameter associated to the filament selected in Slic3r. This is "
+"only supported in recent Marlin."
+msgstr ""
+"该试验参数在E值中使用立方米为单位，而非毫米。如果固件不知道丝料直径，你可以在"
+"起始G代码中输入如'M200 D[filament_diameter_0] T0' 以开启体积模式，并使用在"
+"Slic3r中已选的丝料直径。仅在最新Marlin中支持该功能。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1539
+msgid "Vibration limit (deprecated)"
+msgstr "振动限制(赞成)"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1540
+msgid ""
+"This experimental option will slow down those moves hitting the configured "
+"frequency limit. The purpose of limiting vibrations is to avoid mechanical "
+"resonance. Set zero to disable."
+msgstr ""
+"该试验选项将减缓达到指定频率值的打印动作。目的为避免机械共振。设为0以禁用。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1547
+msgid "Wipe while retracting"
+msgstr "回缩时擦拭"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1549
+msgid ""
+"This flag will move the nozzle while retracting to minimize the possible "
+"blob on leaky extruders."
+msgstr "在回缩时移动喷嘴以避免挤出头滴漏。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1558
+msgid "XY Size Compensation"
+msgstr "XY尺寸补偿"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1560
+msgid ""
+"The object will be grown/shrunk in the XY plane by the configured value "
+"(negative = inwards, positive = outwards). This might be useful for fine-"
+"tuning hole sizes."
+msgstr ""
+"该物体将以指定值（负=往内，正=往外）变大/收缩。对于精确调节孔洞尺寸可能有用。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1566
+msgid "Z offset"
+msgstr "Z补偿"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1567
+msgid ""
+"This value will be added (or subtracted) from all the Z coordinates in the "
+"output G-code. It is used to compensate for bad Z endstop position: for "
+"example, if your endstop zero actually leaves the nozzle 0.3mm far from the "
+"print bed, set this to -0.3 (or fix your endstop)."
+msgstr ""
+"在输出G代码中，所有Z坐标将在原基础上增大（或减少）该数值。用来对糟糕的Z终点挡"
+"板位置进行补偿，如果你的终点挡板为零时，喷嘴离打印热床还有0.3毫米，将该值设"
+"为-0.3（或者调整你的终点挡板）。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1573
+msgid "Z full steps/mm"
+msgstr "Z全部步骤数/毫米"
+
+# "you can calculate this by dividing the number of microsteps configured in your firmware by the microstepping amount (8, 16, 32). "
+# Lost me there.
+# What is "the number of microsteps" and "microstepping amount"?
```

Comment:
Stepper drivers have a fixed multiplier (the common ones support either 16x or 32x).

Each stepper driver, however has a fixed number of full steps per revolution. This setting is to permit automatic adjustment of Z height to increase/decrease Z by the stepper driver's full step. The rest of the text is a way to calculate the number of mm for each full step.